charts/otomi-pipelines/values.yaml

```diff
@@ -22,7 +22,7 @@ tektonTask:
   resources:
     # Tekton Task pod resouce limits and requests
     requests:
-      cpu: '500m'
+      cpu: '50m'
```

Comment:
Are these values also not in the default files? We should also change those.

Reply:
hmm good point. IMHO they should be set via `values/otomi-pipelines/otomi-pipelines.gotmpl`

Reply:
do we want to hardcode them in `otomi-pipelines.gotmpl` or expose them via the `values-schema`?
IMO there is no need to expose them since the user would need to change this if the pipeline can not start and if that is the case the change can not be applied anyway

Reply:
moved resources to otomi-pipelines go template